src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -466,19 +466,27 @@ uct_rc_mlx5_iface_init_rx(uct_rc_iface_t *rc_iface,
     /* MP XRQ is supported with HW TM only */
     ucs_assert(iface->tm.mp.num_strides == 1);
 
-    status = uct_rc_iface_init_rx(rc_iface, rc_config, &iface->rx.srq.verbs.srq);
-    if (status != UCS_OK) {
-        goto err;
-    }
+    if (uct_rc_mlx5_is_devx_srq_supported(md, 0)) {
+        status = uct_rc_mlx5_devx_init_rx(iface, rc_config);
```

Comment:
can do 
`return uct_rc_mlx5_devx_init_rx(iface, rc_config);` here and avoid long else below
would need to move `iface->super.progress = uct_rc_mlx5_iface_progress;`  before the check for devx srq in this case